pennylane_lightning/src/tests/CreateAllWires.cpp

```diff
@@ -0,0 +1,30 @@
+#include "CreateAllWires.hpp"
+namespace Pennylane {
+auto crateAllWires(size_t n_qubits, Gates::GateOperation gate_op, bool order)
```

Comment:
Need to be fixed in `CreateAllWires.hpp` and `est_GateImplementations_CompareKernels.cpp` too
```suggestion
auto createAllWires(size_t n_qubits, Gates::GateOperation gate_op, bool order)
```

Reply:
Yes!